pimcore/lib/Pimcore/Localization/Locale.php

```diff
@@ -111,16 +111,7 @@ public function getDisplayRegions($locale = null)
             $locale = $this->findLocale();
         }
 
-        $regions = [];
-        $locales = $this->getLocaleList();
-        foreach ($locales as $code) {
-            $regionCode = \Locale::getRegion($code);
-            if ($regionCode) {
-                $regionTranslation = \Locale::getDisplayRegion($code, $locale);
-                $regions[$regionCode] = $regionTranslation;
-            }
-        }
-
+        $regions = include(PIMCORE_COMPOSER_PATH . '/umpirsky/country-list/data/' . $locale . '/country.php');
```

Comment:
Unfortunately one thing is missing here, ... we need a check if the `country.php` for the requested locale does exist. if it doesn't we have to fall back on `en`.